src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,74 @@
+package lotto.view;
+
+import lotto.domain.LottoResult;
+import lotto.domain.Prize;
+import lotto.domain.ticket.LottoTicket;
+import lotto.domain.ticket.LottoTickets;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class OutputView {
+    private static final String INFORM_SIZE_MSG = "%d개를 구매했습니다.\n";
+    private static final String WINNING_STATISTICS = "당첨 통계";
+    private static final String DASHES = "---------";
+    private static final String INFORM_RESULT_AND_BONUS_MSG_FORMAT = "%d개 일치, 보너스 볼 일치(%d원) - %d개";
+    private static final String INFORM_RESULT_MSG_FORMAT = "%d개 일치(%d원) - %d개";
+    private static final String INFORM_PROFIT_RATIO_MSG_FORMAT = "총 수익률은 %d%%입니다.";
+    private static final String COMMA = ",";
+    private static final String TICKET_PREFIX = "[";
+    private static final String TICKET_SUFFIX = "]";
+
+    public void printTicketsSize(int size) {
+        System.out.printf(INFORM_SIZE_MSG, size);
+    }
+
+    public void printAllLottoTickets(LottoTickets lottoTickets) {
+        lottoTickets.list().forEach(lottoTicket ->
+                System.out.println(makeEachLottoTicketToString(lottoTicket))
+        );
+    }
+
+    private String makeEachLottoTicketToString(LottoTicket lottoTicket){
+        return lottoTicket.list().stream()
+                .map(Object::toString)
+                .collect(Collectors.joining(COMMA, TICKET_PREFIX, TICKET_SUFFIX));
+    }
+
+    public void printLottoResult(LottoResult lottoResult) {
+        System.out.println(WINNING_STATISTICS);
+        System.out.println(DASHES);
+        printWinningResult(lottoResult);
+        printProfitRatio(lottoResult.calculateProfitPercent());
+    }
+
+    public void printWinningResult(LottoResult lottoResult) {
+        Arrays.stream(Prize.values())
+                .filter(prize -> prize != Prize.LOSING)
+                .sorted(Comparator.comparing(Prize::getRank))
+                .sorted(Comparator.reverseOrder())
+                .forEach(prize ->
+                        System.out.println(makeWinningResultMessage(prize, lottoResult))
+                );
+    }
+
+    public String makeWinningResultMessage(Prize prize, LottoResult lottoResult) {
+        Optional<Long> winningCount = lottoResult.get(prize);
+
+        return String.format(findFormat(prize), prize.getMatchCount(), prize.getPrizeMoney(),
+                winningCount.orElse(0L));
```

Comment:
굳이 위에서 Optional을 먼저 받아오기보다 
```suggestion
                lottoResult.get(prize).orElse(0L));
```
이렇게 하는게 더 가독에 좋을거같아요. 하지만,
위의 언급드린것과는 별개로 Price를 계산하는게 View의 책임으로 보입니다. 당첨이 되지 않았으면 0원을 주는게 뷰의 책임일까요? 혹은 LottoResult의 책임일까요?

Reply:
값을 계산하여 반환하는 것이 LottoResult의 책임이므로, LottoResult에서 하는 것이 더 자연스러운 것 같습니다. Optional을 활용함했으니 책임을 다 한것 아닌가? 잠시 생각했었는데, null일 경우 0으로 만드는 처리를 view가 하고 있으니 이건 아니라는 느낌이 오네요. 어제 Map.getOrDefault() 에 대해 학습했어서, 이렇게 적용해보았습니다!

```java
public class LottoResult{
  ...
  public long get(Prize prize){
    return lottoResult.getOrDefault(prize, 0L);
  }
  ...
}
```

Reply:
오 네 바로 이해하시고 반영하셨군요!! 감사합니다 👍